.github/workflows/php-test.yml

```
@@ -51,11 +51,12 @@ jobs:
         - '7.4'
         - '8.0'
         - '8.1'
+        - '8.2'
         wordpress: [ 'latest' ]
         include:
           - php: '7.4'
             wordpress: '6.3'
-          - php: '8.2'
+          - php: '8.3'
             wordpress: 'latest'
             coverage: true
           - php: '8.3'
```

Shouldn't this now be changed to PHP 8.4, in accordance with what you did for Behat?
```suggestion
          - php: '8.4'
```

---

Yes and no. We run our PHP unit tests in Docker using wp-env, but there is no WP + PHP 8.4 image available yet. See https://github.com/docker-library/wordpress/pull/932